src/webresource-handler/handlers/S3Handler.ts

```diff
@@ -61,6 +105,7 @@ export class S3Handler implements WebResourceHandler {
 		this.maxFileSize = config.maxSize ?? 52428800;
 		this.bucket = config.bucket;
 		this.client = new S3Client(this.config);
+		this.calculateChecksums = config.calculateChecksums ?? true;
```

Comment:
What are the implications on enabling checksum calculations by default? For example, the performance overhead for large files, many files, etc.

Reply:
@pranasziaukas now that's a great question I would also want to know the answer. I suppose that the main risk associated is with higher CPU usage as more and more entities on our DB use webresources. I guess that if proven to be a problem we could change the implementation to be on resource level. For example, we would want checksum calculation for release assets but it might be overkill for logo images... 

For now I would say we are safe with the current behaviour (we are not doing anything like storing the whole file in memory etc) but we do have to pipe all the file content. 

tl;dr: Idk how this will behave at larger scale, but we have alternatives to reduce impact if it happens